src/components/icon/index.jsx

```diff
@@ -0,0 +1,103 @@
+import PropTypes from "prop-types";
+import additionalSVGProps from "./icons/Helpers";
+
+import * as Icons from "./icons";
+
+const COMPONENT_CLASS_NAME = "c-icon";
+
+const Icon = ({ className, context, description, fill, height, name, title, width }) => {
+	const IconName = Icons[name];
+
+	return (
+		<svg
+			className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}
+			width={width}
+			height={height}
+			xmlns="http://www.w3.org/2000/svg"
+			viewBox="0 0 512 512"
+			fill={fill}
+			{...additionalSVGProps(context)}
+		>
+			{context === "image" ? (
+				<>
```

Comment:
@matthewroach I am thinking we may need to differentiate between Icons that are decorative and those that should have meaning.  If they are decorative (in other words, they don't provide any meaning or information), then we should allow the component to add `aria-hidden="true"` attribute.

Reply:
This component handle's both cases - you need to use the `context` property - by default all Icon should be treated as presentational